Blocks/SS/idx_ts.h

```diff
@@ -48,9 +49,38 @@ CURRENT_STRUCT(IndexAndTimestamp) {
   CURRENT_CONSTRUCTOR(IndexAndTimestamp)(uint64_t index, std::chrono::microseconds us) : index(index), us(us) {}
 };
 
+// The same as `IndexAndTimestamp`, but with optional index member.
```

Comment:
For now the `TimestampAndOptionalIndex` is used only to pass head updates to remote subscribers. So it's not really about the _last_ entry or head.. But you are right, there should be more informative comment.